docs/publish/government-cloud-guidance.md

```diff
@@ -24,17 +24,13 @@ For all government cloud (i.e. Sovereign Cloud) environments, the public Microso
 
 [Centralized deployment](/microsoft-365/admin/manage/centralized-deployment-of-add-ins) of add-ins outside of the store is still supported.
 
```

Comment:
Can we define GCC somewhere, especially since the section that had spelt it out is being removed?